sirbot-pyslackers/endpoints/slack/commands.py

```diff
@@ -4,11 +4,14 @@
 from slack import methods
 from slack.events import Message
 
+from .utils import HELP_FIELD_DESCRIPTIONS
+
 LOG = logging.getLogger(__name__)
 
 
 def create_endpoints(plugin):
     plugin.on_command("/admin", tell_admin)
+    plugin.on_command("/command", command_description)
```

Comment:
`/commands` ? Not a fan of the command, I would have prefer a `/help` but I figures it's reserved for slack itself.

Reply:
`help` is a deprecated command for slack itself, and `commands` is apparently a reserved word in the Add Slash Command page.

I'm not a fan of it either, and am very open to alternative suggestions.